app/javascript/controllers/flash_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ['message']
+
+  // Remove flash alerts before 10 seconds.
```

Comment:
Avoid using comments to explain your code; instead, write an auto-explanatory code. In this block of code you could use a variable to explain the amount of time.

```javascript

static timerToHideInSeconds = 10 * 1000;

connect() {
    setTimeout(() => {
      this.messageTarget.remove()
    }, timerToHideInSeconds)
  }
```

something like that.